docs/triage.md

```diff
@@ -2,7 +2,17 @@
 
 As we get more issues and pull requests opened on the GitHub CLI, we've decided on a weekly rotation
 triage role. The initial expectation is that the person in the role for the week spends no more than
-1-2 hours a day on this work; we can refine that as needed.
+1-2 hours a day on this work; we can refine that as needed. Below is a basic timeline for a typical
+triage day.
+
+1. Note the time
+2. Open every new [issue](https://github.com/cli/cli/issues?q=is%3Aopen+is%3Aissue)/[pr](https://github.com/cli/cli/pulls?q=is%3Apr+is%3Aopen+draft%3Afals) in a tab
+3. Go through each one and look for things that should be closed outright (See the PR and Issue section below for more details.)
+4. Go through again and look for issues that are worth keeping around, update each one with labels/pings
+5. Go through again and look for PRs that solve a useful problem but lack obvious things like tests or passing builds; request changes on those
+6. Mark any remaining PRs (ie ones that look worth merging with a cursory glance) as community PRs and move to Needs Review
+7. Look for [issues](https://github.com/cli/cli/issues?q=is%3Aopen+is%3Aissue) and [PRs](https://github.com/cli/cli/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+sort%3Aupdated-desc) updated in the last day and see if they need a response.
```

Comment:
Can we have just "unread/updated issues and PRs"(probably sorted by datetime), instead of "updated in the last day"? If the collaborators are going to bail out after an hour, and there are lots of updates, a few of them may be missed